packages/block-library/src/list/save.js

```diff
@@ -4,10 +4,14 @@
 import { RichText } from '@wordpress/block-editor';
 
 export default function save( { attributes } ) {
-	const { ordered, values } = attributes;
+	const { align, ordered, values } = attributes;
 	const tagName = ordered ? 'ol' : 'ul';
+	const styles = {
+		textAlign: align,
+		listStylePosition: 'inside',
```

Comment:
This probably should be handled in CSS, @jasmussen or @kjellr, any thoughts?

Reply:
In general, yes. I'd love to see us standardizing `alignleft`, `aligncenter`, `alignright` across _all_ blocks. But we're already handling those rules inline on paragraph and heading blocks, so I'm not sure it makes sense to do something different here. 

It might make sense to set up a new class for `listStylePosition`. Maybe something like `has-outside-list` / `has-outside-list`?

Reply:
> In general, yes. I'd love to see us standardizing `alignleft`, `aligncenter`, `alignright` across _all_ blocks.

This is different, it's text alignment, rather than block alignment. I can imagine it's confusing for everyone :)

> But we're already handling those rules inline on paragraph and heading blocks, so I'm not sure it makes sense to do something different here.

It might be not the best approach for 3rd party developers though. See:
https://twitter.com/westonruter/status/1135425519246225409

There is an example shared for Quote block (added in #13248):
https://github.com/WordPress/gutenberg/blob/04ca6b494ddcebc6a78c433c3b85416e6fe64d9b/packages/block-library/src/quote/theme.scss#L16-L28

It might be easier if we would introduce new class names like `has-text-align-left`, `has-text-align-center` and `has-text-align-right`. I can tackle it in a separate PR as work preceding this one.

Reply:
> This is different, it's text alignment, rather than block alignment. I can imagine it's confusing for everyone :)

Oh right — of course. Yeah, it's really confusing. 😄

> It might be easier if we would introduce new class names like has-text-align-left, has-text-align-center and has-text-align-right. I can tackle it in a separate PR as work preceding this one.

That sounds like a great approach. 👍 Thanks!

Reply:
I would echo Kjell that the approach seems sound. It's always nice when we can avoid classes — the fewer, the better — but in cases of explicit choices the classes both make it easy for themes to target specifically, and without high _specificity_. 

The name is also fine, even for RTL, because on RTL it's still an explicit choice to text align left.

Reply:
> The name is also fine, even for RTL, because on RTL it's still an explicit choice to text align left.

This made me think whether we don't flip the value for RTL version. Fix is ready:
https://github.com/WordPress/gutenberg/pull/15964/commits/dac403079c217eeac2c2e41d31a5ce0a210f1abe

😅